import_mu.py

```diff
@@ -343,17 +343,21 @@ def import_mu(self, context, filepath, create_colliders):
         obj.select = False
 
     mu = Mu()
-    if not mu.read(filepath):
-        operator.report({'ERROR'},
-            "Unrecognized format: %s %d" % (mu.magic, mu.version))
-        return {'CANCELLED'}
+    try:
+        if not mu.read(filepath):
```

Comment:
`Mu#read` return `self` and the class does not implement `__nonzero__` so `bool(self)` is always true. So I think this block is actually completely obsolete.